src/Illuminate/Database/Eloquent/ModelNotFoundException.php

```diff
@@ -66,4 +86,24 @@ public function getIds()
     {
         return $this->ids;
     }
+
+    /**
+     * Get the HTTP status code.
+     *
+     * @return int
+     */
+    public function getStatus(): int
+    {
+        return $this->status;
+    }
+
+    /**
+     * Should the response render as a 404?
+     *
+     * @return bool
+     */
+    public function isNotFound(): bool
+    {
+        return $this->status === 404;
```

Comment:
You could use `\Symfony\Component\HttpFoundation\Request::HTTP_NOT_FOUND` here
```suggestion
        return $this->status === Request::HTTP_NOT_FOUND;
```